lib/src/signed_video_authenticity.c

```diff
@@ -260,9 +206,66 @@ authenticity_report_init(signed_video_authenticity_t *authenticity_report)
   authenticity_report->this_version = calloc(1, SV_VERSION_MAX_STRLEN);
 
   latest_validation_init(&authenticity_report->latest_validation);
-#ifdef ACCUMULATED_VALIDATION
   accumulated_validation_init(&authenticity_report->accumulated_validation);
-#endif
+}
+
+static void
+update_accumulated_validation(const signed_video_latest_validation_t *latest,
```

Comment:
Const here is pointless. Parameters are passed by value, therefore you can still change what latest points to (which should violate the const imho), but not the pointer itself in this local context which also isn't meaningful for the latest pointer as it doesn't point to an array.

Reply:
I do not fully agree. Here are my thoughts.
The pointer to `latest`is passed by value hence there is no point to mark that as const. The values in the memory `latest`points to is not and here we tell the compiler that these are not expected to change when calling this function. If so, something is wrong.
Yes, you can always cast const to non const if you want to change a value, but that means you have to do it explicitly. By adding const like above we can more easily catch bugs where values in `latest`change by accident.